Quizzler-iOS13/AppDelegate.swift

```diff
@@ -10,31 +10,32 @@ import UIKit
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
+    
+    var window: UIWindow?
 
     //Version 1
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        let viewController = storyboard.instantiateViewController(identifier: "ViewController") as! ViewController
+        let navigationController = UINavigationController(rootViewController: UIViewController)
+        
+        self.window = UIWindow(frame: UIScreen.main.bounds)
+        self.window?.rootViewController = navigationController
+        self.window?.makeKeyAndVisible()
+        
+ 
         // Override point for customization after application launch.
         return true
     }
 
+    
+    
+    // This function should stay?
```

Comment:
You can delete for now and add only if you'll need it

Reply:
Thank you!